test/commands/user_track/generate_concepts_status_test.rb

```diff
@@ -0,0 +1,177 @@
+require "test_helper"
+
+class UserTrack::GenerateConceptsStatusTest < ActiveSupport::TestCase
+  def setup_user_track
+    track = create :track
+    user_track = create :user_track, track: track
+
+    [track, user_track]
+  end
+
+  def setup_from_factory(track, type, slugs)
+    slugs.map { |slug| create type, slug: slug, track: track }
+  end
```

Comment:
As this method is used exclusively by `setup_concepts` and `setup_concept_exercises`, it's good practice to put it below those methods. It's good to always be able to read a file from top to bottom in terms of execution where possible (it's often impossible to get it exactly right, but working to the spirit of it keeps things clean).